examples/Binance/CoinMFutures.jl

```diff
@@ -39,3 +46,14 @@ Binance.CoinMFutures.income_log(
     endTime = DateTime("2023-07-23"),
     limit = 1000,
 )
+
+Binance.CoinMFutures.order_book(; symbol = "BTCUSD_PERP")
+Binance.CoinMFutures.order_book(; symbol = "BTCUSD_PERP", limit = 10)
+
+Binance.CoinMFutures.premium_index(;
+    pair = "BTCUSD_PERP",
+)
+
+Binance.CoinMFutures.ticker(;
+    pair = "BTCUSD_PERP",
+)
```

Comment:
All public methods must be higher than private ones (before the client is announced)
And add some new examples 
```suggestion
Binance.CoinMFutures.order_book(; symbol = "BTCUSD_PERP")
Binance.CoinMFutures.order_book(; symbol = "BTCUSD_PERP", limit = 10)

Binance.CoinMFutures.premium_index(; pair = "BTCUSD")
Binance.CoinMFutures.premium_index(; symbol = "BTCUSD_PERP")

Binance.CoinMFutures.ticker(; pair = "BTCUSD")
Binance.CoinMFutures.ticker(; symbol = "BTCUSD_PERP")
```